src/test/java/nextstep/subway/acceptance/LineSectionAcceptanceTest.java

```diff
@@ -73,6 +135,67 @@ void removeLineSection() {
         assertThat(response.jsonPath().getList("stations.id", Long.class)).containsExactly(강남역, 양재역);
     }
 
+    /**
+     * Given 지하철 노선에 새로운 구간 추가를 요청 하고
+     * When 지하철 노선의 중간 구간 제거를 요청 하면
+     * Then 노선에 구간이 제거된다
+     */
+    @DisplayName("지하철 노선에 구간을 제거")
+    @Test
+    void removeLineSection_When_중간_역을_제거하면_Then_중간_구간제거() {
+        // given
+        Long 정자역 = 지하철역_생성_요청("정자역").jsonPath().getLong("id");
```

Comment:
```suggestion
        Long 정자역 = 지하철역_생성_요청("정자역")
```

정자역에 대한 아이디를 바로 준다면 어떨까요?